packages/styles/progressbar.css

```diff
@@ -18,3 +20,16 @@
   min-width: var(--space-small);
   transition: width linear var(--progress-bar-animation-timing);
 }
+
+.ProgressBar--thin .ProgressBar--fill {
+  height: var(--space-smallest);
+}
+
+.cauldron--theme-dark {
+  --progress-bar-background-color: var(--accent-dark);
+  --progress-bar-fill-color: var(--accent-info-light);
+}
+
+.cauldron--theme-dark .ProgressBar {
+  border: none;
```

Comment:
This gives the dark themed progress bar a different footprint. It would be better to update `--progress-bar-border-color` to match the background color for dark theme.